scripts/lib/setup/installers.sh

```diff
@@ -191,7 +218,32 @@ function install_android_sdk() {
 }
 
 function install_android_sdk_linux() {
-  cecho "@b@yellow[[+ Skipping Android SDK setup, not implemented on Linux]]"
+  if [ -z "$ANDROID_SDK_ROOT" ]; then
+    if grep -Fq "sdk.dir" $_localPropertiesPath; then
+      local _sdkParentDir="$(awk -F'=' "/^sdk.dir=/{print \$2}" "$_localPropertiesPath")"
+      export_android_sdk_vars $_sdkParentDir
+      cecho "@green[[Android SDK already declared.]]"
+    else
+      local _sdkParentDir=$HOME/Android/Sdk
+      mkdir -p $_sdkParentDir
+      cecho "@cyan[[Downloading Android SDK.]]"
+
+      downloadUrl . sdk-tools-linux.zip https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip && \
```

Comment:
Shouldn't this be using `toolversion` script to get sdk version?

Reply:
yeah, I guess we can put it there.